Framework/API/inc/MantidAPI/Algorithm.h

```diff
@@ -353,10 +353,6 @@ class MANTID_API_DLL Algorithm : public IAlgorithm {
   /// The argument is the property name. Default - do nothing.
   void afterPropertySet(const std::string &) override;
 
-  void filterByProperty(const Kernel::TimeSeriesProperty<bool> & /*filter*/, const std::vector<std::string> &
-                        /* excludedFromFiltering */) override {
-    throw(std::runtime_error("Not yet implmented"));
-  }
```

Comment:
This implementation is no longer needed because I have created a default implementation in the `IPropertyManager` class, so we no longer need to override this function in multiple places. It seems bad to write an implementation in an interface class, because I think ideally all functions in an interface class should be pure virtual, but it looks like `IPropertyManager` doesn't follow that rule anyhow.